lib/api/README.md

```diff
@@ -0,0 +1,129 @@
+## API
+
+This module acts as a javascript wrapper around api.bazaarvoice.com. For more
+information about the API, please see the [API documentation][0].
+
+### Requirements
+
+This module is written using ES6. In order to use the source you may need to
+provide your own transpiler, depending on your supported browsers.
+
+We also take advantage of some built-ins that you may need to polyfill:
+
+- `fetch`: http://caniuse.com/#feat=fetch
+- `Promise`: http://caniuse.com/#feat=promises
+
+#### How to Use
+
+Before using, you'll need to load up the API module:
+
+```javascript
+const api = require('bv-ui-core/lib/api')
+```
+
+Or if you know the specific type of request you are going to make, you can do the following:
+
+```javascript
+const getStatistics = require('bv-ui-core/lib/api/statistics')
+```
+
+## METHODS
+
+Supported methods:
+* [statistics/getStatistics](#getstatistics)
+
+---
+
+### getStatistics
+
+Returns the statistics for a given list of products.
+
+#### Syntax
+
+```javascript
+api.get('statistics', options)
+
+// or
+const getStatistics = require('bv-ui-core/lib/api/statistics')
+getStatistics(options)
+```
+
+#### Parameters
+
+##### options
```

Comment:
This is awesome.  Do you have any good strategies for keeping this in sync with the code? Like what happens when something gets added to `options` - how does a developer know that they should update this also?